packages/apollos-api/src/data/RockConstants.js

```diff
@@ -1,20 +0,0 @@
-import { RockConstants } from '@apollosproject/data-connector-rock';
-
-class dataSource extends RockConstants.dataSource {
-  async createOrFindInteractionComponent({
-    componentName,
-    channelId,
-    entityId,
-  }) {
-    return this.findOrCreate({
-      model: 'InteractionComponents',
-      objectAttributes: {
-        Name: componentName,
-        InteractionChannelId: channelId, // In willow, this field is InteractionChannel instead of Channel for unknown reasons.
```

Comment:
Any reason you deleted this file?

Reply:
Not that I can recall. I think I saw some changes coming in from master and didn't accept them. I have corrected this and added the file back in